Dockerfile

```diff
@@ -1,30 +1,23 @@
-#
-# Compile Go tools and install to /tools/bin
-#
-ARG GO_IMAGE_VERSION='1.17'
 ARG ALPINE_IMAGE_VERSION='3.14'
 
-FROM golang:${GO_IMAGE_VERSION}-bullseye as build
-
-ARG THELMA_VERSION='development'
-
-WORKDIR /build
-COPY . .
-
-# Compile & install runtime dependencies into output/release-assembly
-RUN make release VERSION=${THELMA_VERSION}
-
 #
 # Copy dist into runtime image
 #
-FROM alpine:${ALPINE_IMAGE_VERSION} as runtime
+FROM alpine:${ALPINE_IMAGE_VERSION}
+
+ARG THELMA_LINUX_RELEASE
+
+COPY output/releases/${THELMA_LINUX_RELEASE} .
```

Comment:
Just want to say that this alone is a fantastic speedup! So great to see the binaries re-used instead of another compile! Thank you!